### PR TITLE
New Documentation of yab

### PR DESCRIPTION
**Pull request by lorglas**

### CODE REVIEW

**Comment by humdingerb** [comment]
If you want the package to be rebuild, you have to increase its REVISION.

Furthermore, the SUMMARY seems to be wrong "`An integrated development environment for yab`". Is it the docs for yab or the IDE? In any case, the SUMMARY should mention this being docs.
If it's the docs package for the IDE, the package should probably be renamed to "yab_ide_documentation`.

File: haiku-data/yab_documentation/yab_documentation-1.7.9.recipe [hunk 1]
```diff
@@ -1,13 +1,13 @@
-SUMMARY="An integrated development environment for yab"
-DESCRIPTION="Documentaion for yab"
+SUMMARY="Documentation for the integrated development environment for yab"
+DESCRIPTION="Documentation for yab"
```
Comment (humdingerb) [inline haiku-data/yab_documentation/yab_documentation-1.7.9.recipe:2]: If you change it to "Documentation for the yab IDE" it'll be found when searching for "IDE" in HaikuDepot.
So no new REVISION to force a rebuild of the package?

Comment (bbjimmy) [reply haiku-data/yab_documentation/yab_documentation-1.7.9.recipe:2]: This looks like it is documenting the programming language, not the IDE.

Comment (Begasus) [reply haiku-data/yab_documentation/yab_documentation-1.7.9.recipe:2]: Maybe:
```diff
-SUMMARY="Documentation for yab programming language"
-DESCRIPTION="Documentation for yab"
+SUMMARY="Documentation for yab"
+DESCRIPTION="Documentation for yab programming language and the yab IDE"
```

**Pull request merged**